helm/prereg-application/.gitignore

```diff
@@ -1,2 +1,2 @@
 charts/
-Charts.lock
+Chart.lock
```

Comment:
pre-registration/deploy/prereg
/install.sh
 

CHART_VERSION=0.0.1-develop
PREREG_BOOKING_CHART_VERSION=0.0.1-develop
change to 
CHART_VERSION=1.3.0-develop
PREREG_BOOKING_CHART_VERSION=1.3.0-develop